### PR TITLE
Add doc: how to use custom Kotlin version (Gradle)

### PR DESCRIPTION
**Pull request by GavinRay97**
I don't know how to do this for Maven, I think I've tried once and it was harder than this
If anyone knows, would be great to add for Maven as well

### CODE REVIEW

**Comment by gsmet** [comment]
@glefloch could you have a look at this proposal? Thanks!

File: docs/src/main/asciidoc/kotlin.adoc [hunk 1]
```diff
@@ -303,7 +303,28 @@ tasks.withType<org.jetbrains.kotlin.gradle.tasks.KotlinCompile> {
...
+If you want to use a different version than the one specified by Quarkus' BOM in your application (for example, to try pre-release features or for compatibility reasons), you can do so by using the `strictly {}` version modifier in your Gradle dependencies. For instance:
+
+[source,kotlin]
+----
+plugins {
+    id("io.quarkus")
+    kotlin("jvm") version "1.7.0-Beta"
+    kotlin("plugin.allopen") version "1.7.0-Beta"
+}
+
```
Comment (glefloch) [inline docs/src/main/asciidoc/kotlin.adoc:317]: I think we should leverage resolution for all configurations with the following configuration :

    configurations.all {
        resolutionStrategy {
            force "org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.7.0-Beta"
            force "org.jetbrains.kotlin:kotlin-reflect:1.7.0-Beta"
        }
    }

**Review by glefloch — CHANGES_REQUESTED** [review]
Thanks for this @GavinRay97 I left a comment.

**Comment by glefloch** [comment]
Yes I will apply the fix in the PR

**Comment by GavinRay97** [comment]
Apologies, I didn't notice the reply in my inbox 😬

**Review by glefloch — APPROVED** [review]

**Pull request merged**